bindings/go/constructor/construct_resource_test.go

```diff
@@ -0,0 +1,669 @@
+package constructor
```

Comment:
I am missing a test that contains several resources of the same or different types to make sure that not only one resource per constructor can be processed

Reply:
added